pkg/kubelet/volume_host.go

```diff
@@ -56,6 +58,18 @@ func NewInitializedVolumePluginMgr(
 	plugins []volume.VolumePlugin,
 	prober volume.DynamicPluginProber) (*volume.VolumePluginMgr, error) {
 
+	// Initialize csiDriverLister before calling NewInitializedVolumePluginMgr
+	var csiDriverLister storagelisters.CSIDriverLister
+	const resyncPeriod = 0
```

Comment:
this doesn't seem right, shouldn't it be something >0?

Reply:
I searched for what value resyncPeriod is set to in other places, I saw it was set to 10 min in some places.  I'm not sure what is the reason behind it.  Setting to 10 min now.  Let me know if you have other suggestions.